SplunkAppForWazuh/appserver/controllers/api.py

```diff
@@ -22,6 +22,43 @@
 from log import log
 
 
+# This function change the keys for "********" for don't show the password in the app 
+def cleanKeys(response): 
+    res = response["data"]
+    hide = "********"
+
+    # Remove agent key
+    if "internal_key" in res:
+        res["internal_key"] = hide
+
+    # Remove cluster key (/come/cluster)
+    if "node_type" in res and "key" in res:
+        res["key"] = hide
+
+    # Remove cluster key (/manager/configuration)
+    if "cluster" in res:
+        if "node_type" in res["cluster"] and "key" in res["cluster"]:
+            res["cluster"]["key"] = hide
+    
+    # Remove AWS keys
+    if "wmodules" in res:
+        for wmod in res["wmodules"]:
+            if "aws-s3" in wmod:
+                if "buckets" in wmod["aws-s3"]:
+                    for bucket in wmod["aws-s3"]["buckets"]:
+                        for bKey, bVal in bucket.items():
+                            bVal["access_key"] = hide
+                            bVal["secret_key"] = hide
+
+    # Remove integrations keys
+    if "integration" in res:
+        for integ in res["integration"]:
+            for iKey, iVal in integ.items():
+                iVal["api_key"] = hide
+
+    response["data"] = res
+    return str(response)

```

Comment:
@adri9valle Are you sure that returning a `str` type isn't breaking the `json.dumps()` method below? Please, check that out and also add new mocha tests for this feature. You can still use this branch. Thanks!